apps/tx-signer/env/.env.sample

```
@@ -5,6 +5,7 @@ FUNDING_WALLET_MNEMONIC_V1=
 FUNDING_WALLET_MNEMONIC_V2=
 DERIVATION_WALLET_MNEMONIC_V1=
 DERIVATION_WALLET_MNEMONIC_V2=
-GAS_SAFETY_MULTIPLIER=
+GAS_DEFAULT_MULTIPLIER=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The GAS_DEFAULT_MULTIPLIER key should go before the PORT key

(UnorderedKey)